os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -422,12 +426,11 @@ int8_t cmd_wifi_connect(trwifi_ap_config_s *ap_connect_config, void *arg)
 	wifi_utils_ap_auth_type_e auth = ap_connect_config->ap_auth_type;
 	wifi_utils_ap_crypto_type_e crypto = ap_connect_config->ap_crypto_type;
 	/* WIFI_MANAGER_CRYPTO_AES: 3 */
-	if (crypto == 3 && auth != WIFI_UTILS_AUTH_WPA2_PSK) {
+	if (crypto == 3 && ((auth != WIFI_UTILS_AUTH_WPA2_PSK) && (auth != WIFI_UTILS_AUTH_WPA3_PSK))) {

```

Comment:
Is this correct to check `(auth != WIFI_UTILS_AUTH_WPA2_PSK) && (auth != WIFI_UTILS_AUTH_WPA3_PSK)`? not `(auth != WIFI_UTILS_AUTH_WPA2_PSK) || (auth != WIFI_UTILS_AUTH_WPA2_PSK)`?
I think `auth` cannot be `WIFI_UTILS_AUTH_WPA2_PSK` and `WIFI_UTILS_AUTH_WPA2_PSK` simultaneuosly.

Reply:
Sorry I don't get what you mean. Is there a type error in you comment? My logic is as below, when  cypto is 3, which is aes, auth should be WPA2_PSK or WPA3_PSK, if not, return error

Reply:
Oh sorry. This is correct. I was confused. Let's keep this.

Reply:
No problem~